english/batman/batman.md

```diff
@@ -29,3 +29,4 @@ Uncle Ben: With great power comes great responsibility.
 (Hey Uncle Ben was a character in Spiderman!!, doesn't belong here)
 
 The real hero is alfred the butler..
+Who is really [Luke Skywalker's](http://fellowshipoftheminds.com/2015/11/23/star-wars-goes-to-illuminati-dark-side/) son!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.